--- a/epan/epan.h	2024-04-20 00:34:15.534364916 -0700
+++ b/epan/epan.h	2024-04-20 00:34:25.094467480 -0700
@@ -10,9 +10,6 @@
 #ifndef __EPAN_H__
 #define __EPAN_H__
 
-#ifdef __cplusplus
-extern "C" {
-#endif /* __cplusplus */
 
 #include <glib.h>
 #include <epan/tvbuff.h>
@@ -22,6 +19,9 @@
 #include <epan/register.h>
 #include "ws_symbol_export.h"
 
+#ifdef __cplusplus
+extern "C" {
+#endif /* __cplusplus */
 
 /** Global variable holding the content of the corresponding environment variable
  * to save fetching it repeatedly.
